dev/release/changelog.py

```diff
@@ -84,7 +84,8 @@ def format_changelog_website(issues, out):
         'Wish': NEW_FEATURE,
         'Task': NEW_FEATURE,
         'Test': BUGFIX,
-        'Bug': BUGFIX
+        'Bug': BUGFIX,
+        'Sub-task': NEW_FEATURE
```

Comment:
Not sure we want to include subtasks in the issues list.

Reply:
@xhochy , I wasn't sure either but when building the changelog through changelog.py script, it failed saying Sub-Task issue category not found. I think the reason is that when script pulled 0.9.0 items from JIRA, some of them had issue type as Sub-Task which was not a recognized category in the script. So I just included it.